.github/workflows/prettier.yml

```diff
@@ -0,0 +1,26 @@
+name: Check code format
+
+on:
+  pull_request:
+
+jobs:
+  prettier:
+    runs-on: windows-latest
+    defaults:
+      run:
+        working-directory: frontend
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.16.1
+
+      - name: Check code format
+        run: |
+          npm ci
```

Comment:
Here maybe it would be more clean to have a separate action for `npm ci` vs `npm run format:check` so that in the github action tab we have a clean output of file that need formatting (and that doesn't start with all the logs from `npm ci`)